coresdk/src/coresdk/database.h

```diff
@@ -1,60 +1,301 @@
-//
-//  database.h
-//  splashkit
-//
-//  Created by Jake Renzella on 2/08/2016.
-//  Copyright © 2016 Andrew Cain. All rights reserved.
-//
+/**
+ * @header Database
+ * @author Jake Renzella
+ * @brief SplashKit Database allows you to create, query and otherwise manipulate a database.
```

Comment:
Try and leave comments within 80 characters. I'll show you how to set up in Xcode tomorrow :)